src/Void.Core/CommandMode.fs

```diff
@@ -15,23 +16,40 @@ module CommandMode =
     let private requestLanguageForInterpreting buffer =
         buffer, GetCurrentCommandLanguageRequest :> Message
 
-    let handleGetCurrentCommandLanguageResponse buffer response =
+    let private interpretFragment buffer maybeResponse =
         {
-            Language = response.CurrentCommandLanguage
-            Fragment = !buffer
-        } :> Message
+            Language =

```

Comment:
Hm, does seem like there should be another way to do this. However I'm always torn because pattern matching on a maybe is fairly clean.